lib/content_caching/adapters/aws.rb

```diff
@@ -33,17 +33,20 @@ def delete document_path
       private
 
       def service
-        @service ||= S3::Service.new(
-          :access_key_id => self.options[:aws_access_key_id],
-          :secret_access_key => self.options[:aws_secret_access_key],
-          :use_ssl => true,
-          :use_vhost => true
+        @service ||= ::Aws::S3::Resource.new(
+          credentials: aws_credentials,
+          region: self.options[:region]
         )
       end
 
       def bucket
         @bucket ||= service.bucket(self.options[:directory])
       end
+
+      def aws_credentials
+        ::Aws::Credentials.new self.options[:aws_access_key_id],
```

Comment:
please use () , we are using there everywhere, let's keep it that way

Reply:
Noted. Thanks.